client/client.go

```diff
@@ -188,6 +214,284 @@ func (q Query) AddResponse(resp *http.Response) {
 	}
 }
 
+// Request processing
+
+// ExecuteWebsocketQuery handles Websocket queries
+func ExecuteWebsocketQuery(query Query) {
+	url := query.URL()
+	c, resp, err := websocket.DefaultDialer.Dial(url, query.Headers())
+	if query.CheckErr(err) {
+		return
+	}
+	defer c.Close()
+	query.AddResponse(resp)
+	messages := query["messages"].([]interface{})
+	for _, msg := range messages {
+		err = c.WriteMessage(websocket.TextMessage, []byte(msg.(string)))
+		if query.CheckErr(err) {
+			return
+		}
+	}
+
+	err = c.WriteMessage(websocket.CloseMessage,
+		websocket.FormatCloseMessage(websocket.CloseNormalClosure, ""))
+	if query.CheckErr(err) {
```

Comment:
Same as above? Please, check elsewhere.

Reply:
In general, I added log statements to the new gRPC code where it would help me debug the code. I didn't add them everywhere. I suggest we add log statements as we need them, not dogmatically everywhere.